docs/developers_guide/organization.md

```diff
@@ -2022,3 +2022,34 @@ A developer  defines a test suite by creating a `.txt` file within the
 `polaris/<component>/suites` directory.  The format of the `.txt` file is a 
 list of the work directories to  the tests desired to be part of the suite.  A 
 line starting with `#` will be  treated as a comment line.
+
+The philosophy and requirements for the test suites are as follows:
+
+### Pull-Request (PR) suite
+
+The PR suite is intended to be run in the context of its namesake, a pull-
+request to the model component. It should require no more than 32 cores (the
+minimum number of cores on a typical compute node) and take no more than 20
+minutes of run time on 32 cores.
```

Comment:
I think we want to rewrite this with task parallelism in mind.  We want the suite to require no more than 256 cores, and it is preferable if individual tests only use about 32 cores, but more are okay if necessary.  Currently, the QU60 cosine bell test requires 56 cores (assuming cosine bell would be part of the PR suite)

I'll also note that Cori was the machine with 32-core nodes (Compy and Anvil have 36, Perlmutter, Chyrsalis and Chicoma have 128).  It seems like running on 2 to 4 Chrysalis, Perlmutter or Chicoma nodes (targeting 2 but using 4 for quicker testing) would be a good plan.

Reply:
@xylar Good point. I updated the text. Let me know if that reflects what you had in mind.